elements/dash-checkbox/dash-checkbox.js

```diff
@@ -103,6 +103,11 @@
      */
     set disabled(isDisabled) {
       this.setAttribute('aria-disabled', isDisabled);
+      if (isDisabled) {
```

Comment:
So `attributeChangedCallback` gets called for all attributes on boot? Why do you handle any in `connectedCallback` then?

Maybe add a test for this anyway? i.e.
```html
<dash-checkbox aria-disabled="true">  <!-- assert this is not focusable -->
```